library/core/functions.general.php

```diff
@@ -3110,6 +3110,32 @@ function safeImage($ImageUrl, $MinHeight = 0, $MinWidth = 0) {
     }
 }
 
```

Comment:
> There is a difference between URLmatch and the preg_match thought! I don't know if we can safely swap one for the other. TBD

There is a difference between the two functions, which is why I pulled it out into its own function. The `isTrustedDomain()` function doesn't match `*` wildcards, whereas the `matchTrustedDomain()` function does.